akka-projection-jdbc/src/main/resources/reference.conf

```diff
@@ -5,6 +5,7 @@
 akka.projection.jdbc {
   # choose one of: mysql-dialect, postgres-dialect, mssql-dialect, oracle-dialect or h2-dialect (testing)
   dialect = ""
+  dispatcher = "akka.projection.jdbc.blocking-jdbc-dispatcher"
```

Comment:
the  naming convention we  have used in several other places for this kind of property is `use-dispatcher`

Reply:
Done ✅